lib/signs_ui_web/controllers/messages_controller.ex

```diff
@@ -10,7 +10,7 @@ defmodule SignsUiWeb.MessagesController do
 
   def create(conn, params) do
     case Messages.add_message(params) do
-      {:ok, messages} ->
+      {:ok, _messages} ->
```

Comment:
what's this part?

Reply:
there was a compile warning so  i guess i drive by'd it.  i can do it separately if youd prefer.

Reply:
nope that's fine.